packages/remix/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@clerk/clerk-remix",
+  "version": "3.0.0-alpha.0",
```

Comment:
@igneel64 @SokratisVidros Let's figure out the version early tomorrow

Reply:
We have `clerk-js`, `clerk-react`, `clerk-expo` & `next`. I'd stick to the `clerk-remix` at it's hard to migrate the rest. @colinclerk WDYT?

Reply:
FYI: https://docs.npmjs.com/cli/v8/commands/npm-deprecate

Reply:
I am all in for the shorter one, let's schedule the deprecation for a few more e.g. clerk-sdk-node